docs/sql/joins.txt

```diff
@@ -0,0 +1,161 @@
+.. highlight:: psql
+.. _sql_joins:
+
+Joins
+=====
+
+Cross Joins
+-----------
+
+Referencing two tables results in a ``CROSS JOIN``.
+The result contains every possible combination (`Cartesian Product`_)
```

Comment:
The result is computed by creating every possible combination ... and then applying the given query operation on it (WHERE clause, SELECT list, ORDER BY clause, ...):